src/persist-client/examples/maelstrom/node.rs

```diff
@@ -125,7 +125,7 @@ where
             };
 
             let service = Arc::clone(&self.service);
-            let _ = mz_ore::task::spawn(|| "maelstrom::handle".to_string(), async move {
+            mz_ore::task::spawn(|| "maelstrom::handle".to_string(), async move {
```

Comment:
It's https://rust-lang.github.io/rust-clippy/master/index.html#let_underscore_future.

Reply:
> Futures must be polled for work to be done.

ah, what an odd interaction between the lint and the one case this isn't true

Reply:
oh you had this in the PR description 🤦, sorry!

Reply:
All good!

There is some discussion here, in case you haven't seen it: https://github.com/rust-lang/rust-clippy/issues/9932. I would be fine with simply disabling this lint, if you'd rather keep the `let _ =`. Although it generally seems to be a good thing to have. I think the other lint that prevents us from forgetting to `await` futures is the one that checks for `#[must_use]`, and that is disabled by assigning the return value.

Reply:
yeah, I was just reading that. it was indeed here for exactly the reason someone in that thread points out, but it's marginal enough that I don't particularly care

> The let _ is indeed not strictly necessary (JoinHandle isn't must_use), however I do find it serves a legitimate documentation goal, it signals to the reader that the return value of spawn is intentionally dropped, whereas with just spawn there's a doubt left of whether the original developer simply forgot, or not.
> 
> It can be handled differently: comments, calling mem::drop, ... let _ = is the idiomatic way of signalling "I don't care about the return value", though.